dataland-frontend/tests/e2e/fixtures/common/DataPointFixtures.ts

```diff
@@ -1,5 +1,10 @@
 import { faker } from "@faker-js/faker";
-import { CompanyReportReference, DataPointBigDecimal, DataPointYesNo, QualityOptions } from "@clients/backend";
+import {
+  CompanyReportReference,
```

Comment:
The method `generateYesNoOrEmptyDatapoint` is unnecessary.
We can remove it and directly use `valueOrUndefined(generateDatapoint(randomYesNo(), reports))` wherever we need it.

Reply:
I think these are from the generated files from the devtools. Should we then modify the generated lines of code or keep it like they are now?

Reply:
Yes this method is used inside of the auto-generated fixture-helper files in the devtools.
Therefore this would need to change there too.
Therefore let's leave it for another PR.   Too much effort now to touch the devtools again, and a little out of scope.

Reply:
Ok